.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2024 Intel Corporation
+
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
```

Comment:
Are these steps applicable to reproduce the issue/behavior?

Reply:
Yes. In case someone reports the bug I am saying provide the steps.

Reply:
Should we use different text to avoid confusions? For example:
```
1. Configure `gnbsim`
2. Launch `gnbsim`
3. Etc.
```
If think think, the current text is clear, please let me know and I can approve the PR

Reply:
Yes pls